src/test/regress/expected/local_shard_copy.out

```diff
@@ -0,0 +1,494 @@
+CREATE SCHEMA local_shard_copy;
+SET search_path TO local_shard_copy;
+SET client_min_messages TO DEBUG;
+SET citus.next_shard_id TO 1570000;
+SELECT * FROM master_add_node('localhost', :master_port, groupid := 0);
+DEBUG:  schema "public" already exists, skipping
```

Comment:
This test randomly fails:
```
+ diffs=/home/circleci/project/build-11/src/test/regress/regression.diffs
+ test -f /home/circleci/project/build-11/src/test/regress/regression.diffs
+ cat /home/circleci/project/build-11/src/test/regress/regression.diffs
--- /home/circleci/project/src/test/regress/expected/local_shard_copy.out.modified	2020-03-23 07:26:53.380983908 +0000
+++ /home/circleci/project/build-11/src/test/regress/results/local_shard_copy.out.modified	2020-03-23 07:26:53.388983882 +0000
@@ -8,24 +8,29 @@
 DEBUG:  extension "plpgsql" already exists, skipping
 DETAIL:  NOTICE from localhost:57636
 DEBUG:  schema "citus_mx_test_schema" already exists, skipping
 DETAIL:  NOTICE from localhost:57636
 DEBUG:  schema "citus_mx_test_schema_join_1" already exists, skipping
 DETAIL:  NOTICE from localhost:57636
 DEBUG:  schema "citus_mx_test_schema_join_2" already exists, skipping
 DETAIL:  NOTICE from localhost:57636
 DEBUG:  schema "citus_mx_schema_for_xacts" already exists, skipping
 DETAIL:  NOTICE from localhost:57636
+DEBUG:  schema "local_shard_execution" already exists, skipping
+DETAIL:  NOTICE from localhost:57636
 NOTICE:  Replicating reference table "customer_mx" to the node localhost:57636
 NOTICE:  Replicating reference table "nation_mx" to the node localhost:57636
 NOTICE:  Replicating reference table "part_mx" to the node localhost:57636
 NOTICE:  Replicating reference table "supplier_mx" to the node localhost:57636
+NOTICE:  Replicating reference table "reference_table" to the node localhost:57636
+DEBUG:  table "reference_table_1470000" does not exist, skipping
+DETAIL:  NOTICE from localhost:57636
  master_add_node 
 -----------------
               32
 (1 row)
 
 SET citus.shard_count TO 4;
 SET citus.shard_replication_factor TO 1;
 SET citus.replication_model TO 'streaming';
 CREATE TABLE reference_table (key int PRIMARY KEY);
 DEBUG:  CREATE TABLE / PRIMARY KEY will create implicit index "reference_table_pkey" for table "reference_table"

======================================================================

+ exit 2

Exited with code exit status 2
CircleCI received exit code 2
```

@SaitTalhaNisanci would you mind looking into this briefly and understand why? If that's something not related to the correctness -- which seems so, maybe simply change the log level

Reply:
I believe this might solve it: https://github.com/citusdata/citus/pull/3635

if not, I might change the log level as you suggested, it is not related to correctness I think.